node/cn/backend.go

```diff
@@ -441,6 +442,38 @@ func New(ctx *node.ServiceContext, config *Config) (*CN, error) {
 	cn.addComponent(cn.ChainDB())
 	cn.addComponent(cn.engine)
 
+	// migrate gov DB
```

Comment:
Can we move it into gov.Init()? The timing isn't too different.

Reply:
fixed 11d7ee7